lib/document/mongoDbRepository/getDocumentDatabaseFactory.js

```diff
@@ -7,12 +7,12 @@
 function getDocumentDatabaseFactory(connectToDocumentMongoDB, documentMongoDBPrefix) {
   /**
    * @typedef getDocumentDatabase
-   * @param {string} dataContractId
+   * @param {Buffer} dataContractId
    * @return {Db}
    */
   async function getDocumentDatabase(dataContractId) {
     const documentMongoDBClient = await connectToDocumentMongoDB();
-    return documentMongoDBClient.db(`${documentMongoDBPrefix}${dataContractId}`);
+    return documentMongoDBClient.db(`${documentMongoDBPrefix}${dataContractId.toString()}`);
```

Comment:
We need to make sure that `dataContractId` is an `Identifier` (`new Identifier(dataContractId).toString()`)